Refactor getting started guide to use the sample spring-petclinic (#1085)
redhat-developer/service-binding-operator: docs/userguide/modules/getting-started/partials/creating-db-pgcluster.adoc

== Creating a PostgreSQL database instance

> This scenario uses a PostgreSQL database service, which you must install using an Operator Lifecycle Manager (OLM).

Reviewer: ```suggestion
This scenario uses a PostgreSQL database service, which you must install using the Operator Lifecycle Manager (OLM).
```

Author: thx

[discrete]
.Prerequisites
* You have installed the https://operatorhub.io/operator/postgresql[Crunchy Postgres Operator
from Crunchy Data available at OperatorHub.io] and the Operator is available in the `{quickstart-namespace}` namespace.

The installation of the Crunchy PostgreSQL Operator does not create a database instance. To create a database service instance, you must create the following custom resource (CR), which will be processed by the Operator:

+++ <details><summary> +++
`PostgresCluster` resource to run the actual database instance
+++ </summary><div> +++
[source,yaml]
----
include::attachment$pgcluster-deployment.yaml[tag=db-deployment]
----
+++ </div></details> +++

[discrete]
=== Procedure

> . To create a database instance, create a `PostgresCluster` custom resource by running the following command:

Reviewer: ```suggestion
. To create a database instance, create a `PostgresCluster` CR by running the following command:
```

Author: thx

[source,bash,subs="attributes,macros"]
+
----
kubectl apply -f link:{attachmentsdir}/pgcluster-deployment.yaml[pass:a[{site-url}/{page-component-name}/{page-module}/{attachmentsdir}/pgcluster-deployment.yaml]] -n {quickstart-namespace}
----
+

> . After the database is created, ensure that all the pods in the `{quickstart-namespace}` namespace are running (it will take a few minutes):

Reviewer: ```suggestion
. After the database is created, verify that all the pods in the `{quickstart-namespace}` namespace are running (it will take a few minutes):
```

Author: thx

+
[source,bash,subs="attributes,macros"]
----
kubectl get pods -n {quickstart-namespace}
----
+

> .You should see an output similar to the following example:

Reviewer: ```suggestion
.Example output
```

Author: thx
